### ISSUE TITLE
Introduce Organization > PublicOrganization

### ISSUE THREAD

**Issue by tfrancart** [schemaorg/schemaorg#1786]
SDO has "GovernmentOrganization"; this is too restrictive to be able to be used as a range for e.g. legislationPassedBy or legislationResponsible. However a "PublicOrganization" would do. Not all public organizations can be said to be "governmental" organizations.

See the Core Public Organization Vocabulary (CPOV) for inspiration : https://joinup.ec.europa.eu/release/core-public-organisation-vocabulary-v100. 

(this is a nice-to-have, opened for discussion, but nothing blocking for #1743)

**Comment by danbri** [comment]
Can you suggest a definition?

**Comment by tfrancart** [comment]
[CPOV specification](https://joinup.ec.europa.eu/rdf_entity/http_e_f_fdata_ceuropa_ceu_fw21_f73920f8b_b6c65_b4c82_bbe8c_b23a6d2cc4d58/releases) has a nice one : "Any organization that is defined as being part of the public sector by a legal framework at any level." (also discussed [here](https://joinup.ec.europa.eu/discussion/definition-public-organisation)).

Want a more detailled one from the PSI Directive ?

> "The State, regional or local authorities, bodies governed by public law and associations formed by one or several such authorities or one or several such bodies governed by public law”. [The PSI directive] further defines a body governed by public law as any body “(a) established for the specific purpose of meeting needs in the general interest, not having an industrial or commercial character; and (b) having legal personality; and (c) financed, for the most part by the State, or regional or local authorities, or other bodies governed by public law; or subject to management supervision by those bodies; or having an administrative, managerial or supervisory board, more than half of whose members are appointed by the State, regional or local authorities or by other bodies governed by public law."

The URI in CPOV is http://data.europa.eu/m8g/PublicOrganisation

**Comment by camfindlay** [comment]
+1

**Comment by thadguidry** [comment]
@tfrancart that is not a definition of "Public Organization" but instead a PSI directive describing a "public sector body"
- Article 2 - Definitions
For the purpose of this Directive the following definitions shall apply:
1. "public sector body" means the State, regional or local authorities, bodies governed by public law and associations formed by one or several such authorities or one or several such bodies governed by public law;

I would define "Public Organization" with a broader acceptable definition that also aligns with the [PSI directive of Article 2, number 2 (a)](http://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32003L0098&qid=1456507093478&from=EN) a body governed by public law "established for the specific purpose of meeting needs in the general interest, not having an industrial or commercial character", of which Phil Archer and team seemed to say was OK as well https://joinup.ec.europa.eu/discussion/definition-public-organisation#

**Comment by tfrancart** [comment]
@thadguidry : I proposed 2 definitions, the first of which already refers to the discussion you mention; it is extracted from the final version of the CPOV ontology.
I don't have a strong opinion on the definition, as long as this new type eventually makes its way to the stable version of SDO.

**Comment by thadguidry** [comment]
@tfrancart Understood, but we have to be a bit careful with the notion of the term "public" in any Type within Schema.org.  That term has legal and humane implications and it throws things into slightly different context when used outside the USA.  This is something that is similar to some types we have within Medicine, and other domains.

Anyways, let's compare the differences on the 2 definitions being proposed and come to likes or dislikes on any bullet points:

1. Any organization that is defined as being part of the public sector by a legal framework at any level.

A. doesn't mention "public law", but "legal framework". **This is broader.**
B. introduces "public sector" which is a slightly slippery slope in many countries. **Possible caveat.**
C. uses organization as a parent concept, rather than body.
D. has a part of relationship to public sector rather than within public sector. **This is broader.**
E. makes no mention if it could be private or even have some commercial interests. **This is broader.**

2. a body governed by public law established for the specific purpose of meeting needs in the general interest, not having an industrial or commercial character.

A. must be defined by "public law". **This is narrower**
B. uses body, which includes more than organization's. Could be tamed by switching to "organization".
C. says it cannot have commercial interests. **This is narrower**

All the above makes no mention of any "profit" status, and I'm NOT OK with that for the concept of a "Public Organization".  Of course any "profit" status can be captured with a property, but I strongly prefer having a distinct type for "NonProfitOrganization" to capture particulars.

**Comment by tfrancart** [comment]
I still think this would be a valuable addition to schema.org. @danbri, @RichardWallis : can we keep this opened, or move this to the other repo ? thanks

**Comment by jmarks** [comment]
i agree. It would be good to connect a site, page or data with the W3C DID
of the publisher.

**Comment by danbri** [comment]
I think this has potential but would benefit from more discussion including from folk in public sector data efforts of various countries. We can continue discussion in the other repo.

**Comment by RichardWallis** [comment]
See issue schemaorg/schemaorg#7 for the context of the move from the main Schema.org [issue tracker](https://github.com/schemaorg/schemaorg/issues) to this repository.
